### ISSUE TITLE
MultiRMSE with categorical features

### ISSUE THREAD

**Issue by Metamorphus** [catboost/catboost#1112]
Problem:
catboost version: 0.20.1
Operating System: Ubuntu 18.04.1
CPU: Intel(R) Xeon(R) Platinum 8124M CPU @ 3.00GHz

```import numpy as np
from catboost import Pool, CatBoostRegressor

X = np.random.randint(10, size=(10,10))
y = np.random.random(size=(10,7))

pool = Pool(
    X, 
    y, 
    cat_features=[0,1,4]
)

model = CatBoostRegressor(num_trees=100, 
                          depth=7, 
                          learning_rate=0.05,
                          loss_function='MultiRMSE')

model.fit(pool)
```

`Segmentation fault (core dumped)`

Removing categorical features and/or changing loss to RMSE with changing labels to 1D fixes the issue.

**Comment by annaveronika** [comment]
categorical features are not supported for multi regression, we'll fix the segfault soon, but the support will come out later

**Comment by annaveronika** [comment]
Segfault is fixed

**Comment by andrey-khropov** [comment]
Release r0.20.2 reports an error instead of Segfault.

**Comment by kizill** [comment]
We've implemented this in 0.24.2, closing as solved 😺
